### DISCUSSION TITLE
Two accepted discounts at the same time

### DISCUSSION THREAD

**Discussion by dangtiendung1201** [community/community#59688] [GitHub Education] [answered]
### Hi everyone,

Hi everyone.

I got 2 GitHub Education Pack at the same time.
![image](https://github.com/community/community/assets/53939861/98b8d3d6-5dd3-4259-b5e6-f3ced1ea4579)
One of them has expired, the other has just been renewed this year.  But GitHub charges me 4 dollars for using Pro plan next month.
![image](https://github.com/community/community/assets/53939861/3909dda5-03ea-4f27-928d-820acd3ab4d5)

Is this a bug? Please notice this.

Thanks in advance.

**Comment by dangtiendung1201** [comment]
I have just received this email from GitHub.
![image](https://github.com/community/community/assets/53939861/2c5dc25b-402c-4660-abf1-7ad770c5d554)

**Comment by dangtiendung1201** [comment]
Moreover, I have trouble with renewing / reverifying the expired one. When I use my phone, GitHub only use my front camera instead of the rear.

**Comment by samus-aran** [comment, marked as answer]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
